en/flight_modes/mission.md

```diff
@@ -30,7 +30,11 @@ At high level all vehicle types behave in the same way when MISSION mode is enga
    > **Note** If you have a *Jump to item* command in the mission, moving to another item will **not** reset the loop counter. One implication is that if you change the current mission command to 1 this will not "fully restart" the mission.
 1. The mission will only reset when the vehicle is disarmed or when a new mission is uploaded.
 
-Missions can be paused by activating [HOLD mode](../flight_modes/hold.md). The mission will then continue from the current mission command when you reactivate the MISSION flight mode.
+> **Tip** To automatically disarm the vehicle after it lands in mission mode, in QGroundControl go to [**Vehicle Setup > Safety**](https://docs.qgroundcontrol.com/en/SetupView/Safety.html), navigate to "Land Mode Settings" and check the box labelled "Disarm after". Enter your desired wait time before you want to disarm.

```

Comment:
FYI, there is a bug in gitbook that means the rendering of adjacent note/tip/warning types is poor. The way to fix this is to add `<span></span>` immediately before the second item.